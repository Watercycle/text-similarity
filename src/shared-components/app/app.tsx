import { Component } from 'solid-js'
import { PrimaryPage } from '../../pages/primary-page/primary-page'
import styles from './app.styl'

export interface Props {}

export const App: Component<Props> = props =>
    <PrimaryPage/>
