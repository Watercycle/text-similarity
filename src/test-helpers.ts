import * as dom from 'solid-js/dom'

export function render(jsx: JSX.Element): string {
  const div = document.createElement('div')

  const dispose = dom.render(() => jsx, div)
  dispose()

  return div.children[0].outerHTML;
}

export function assertMatchesSnapshot(jsx: JSX.Element) {
  const html = render(jsx);
  expect(html).toMatchSnapshot()
}
