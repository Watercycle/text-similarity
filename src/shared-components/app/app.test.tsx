import { assertMatchesSnapshot } from "../../test-helpers"
import { App } from './app'

it('works', () => {
    assertMatchesSnapshot(() => <App>Yeet</App>)
})
