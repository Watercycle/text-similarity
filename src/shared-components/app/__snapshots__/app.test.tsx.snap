// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `"<div class=\\"root\\">Hello World!</div>"`;
